src/app/pages/registration/registration-form/registration-form.component.html

```diff
@@ -5,99 +5,74 @@ <h2>Registration Form</h2>
 
   <div class="row-1140 pt50">
     <div class="alert alert-danger" role="alert" *ngIf="error"><strong>Motyla Noga..</strong> {{error}}</div>
-    <form (ngSubmit)="save()" #registrationForm="ngForm" novalidate *ngIf="model">
-      <div class="form-group">
-        <label for="buyer">original buyer</label>
-        <input type="text" id="buyer" class="form-control" [value]="model.originalBuyer" disabled>
-
-      </div>
-      <div class="form-group required">
-        <label for="name">full name</label>
-        <input type="text" id="name" class="form-control" [(ngModel)]="model.name" #name="ngModel" name="name" required
-               maxlength="255">
-        <div class="alert alert-danger" role="alert" *ngIf="name.errors?.required && (name.touched || submitted)">
+    <form (ngSubmit)="save()" [formGroup]="registrationForm" *ngIf="registrationForm">
+      <mat-form-field class="full-width">
+        <input matInput placeholder="voucher" type="text" id="voucher2"
+             name="voucher" formControlName="voucher"
+             maxlength="255">
+      </mat-form-field>
+      <mat-form-field>
+        <input type="text" id="city" placeholder="city" formControlName="city"
+             matInput name="city"
+             required maxlength="255">
```

Comment:
I am wondering if we can ask this data now (GDPR). We dont need this data to do our business, so for me it is questionable.

Reply:
lets talk about it on todays call